### ISSUE TITLE
Updated for 4.0

### ISSUE THREAD

**Issue by decryptedchaos** [godot-extended-libraries/godot-realistic-water#6]
I have, more or less updated this to work with Godot 4.0   I still have a few quirks I'm working out but its more or less functional.  

Let me know if you'd like a pull request

**Comment by joshchristiane** [comment]
I would love one, thanks!

**Comment by azimuthdeveloper** [comment]
Do you have a branch that works with 4.0, and if you do, can you link to it?

EDIT: Just found it :https://github.com/decryptedchaos/godot-realistic-water

**Comment by decryptedchaos** [comment]
> Do you have a branch that works with 4.0, and if you do, can you link to it?

Yes,  keep in mind its not fully functional, and i think it the shader might need more tweaking for 4.0 stable   i had it half working with beta 10  but that was some time ago now.
